browser/BUILD.gn

```diff
@@ -278,6 +280,21 @@ source_set("browser_process") {
     ]
   }
 
+  if (gemini_enabled) {
```

Comment:
this should go in browser/gemini/BUILD.gn - in general every folder should have its own BUILD.gn to avoid having one big BUILD.gn with a confusing mess of deps